kubespawner/spawner.py

```diff
@@ -1127,3 +1216,34 @@ def get_args(self):
                 args[i] = '--hub-api-url="%s"' % (self.accessible_hub_api_url)
                 break
         return args
+
+    def _options_form_default(self):
```

Comment:
Can you add docstrings for this method?

Reply:
Sure ! By the way, do you know how the magic work behind the [_]options_form*? I have no idea how it get called (by tornado?)

Reply:
It's a JupyterHub feature, see https://github.com/jupyterhub/jupyterhub/blob/a58bea6d933b7bc0bcfe4b08dfff4f31dd2b5909/jupyterhub/spawner.py#L271 and related.

Reply:
Yes but the call is for the function self.options_form and ultimately the _<functionname>_default is called...